### ISSUE TITLE
VAtools vcf-expression-annotator does not add gene expression for genes without annotated gene names

### ISSUE THREAD

**Issue by bryanfisk** [griffithlab/VAtools#52]
VAtools vcf-expression-annotator won't add gene expression values to vcf entries that occur on a gene that has only a gene id and does not have an annotated gene name. Example inputs included.
[annotated.bam_readcount.vcf.gz](https://github.com/griffithlab/VAtools/files/7375086/annotated.bam_readcount.vcf.gz)
[gene_abundance.tsv.gz](https://github.com/griffithlab/VAtools/files/7375089/gene_abundance.tsv.gz)

**Comment by malachig** [comment]
I have recently encountered this issue in looking at results in pVACview.  I was surprised to see a gene with a blank (empty) expression value.  When I looked up that gene in the kallisto gene expression file from the pipeline I realized that it does have a gene expression value, but it does not have a gene name.  

So, while the annotation reports the gene using the gene ID, we don't get the gene expression value for that gene because it doesn't have a name.  

In general, using gene names when mapping information is problematic for Ensembl because: (a) some Ensembl genes don't have names, (b) the same Ensembl gene name can be used multiple times.  

Only the gene IDs are guaranteed to be present and unique for Ensembl (as infuriating as this is, it appears to be the case for many Ensembl annotation sets).
